torch/nn/functional.py

```diff
@@ -401,13 +400,17 @@
         averaging calculation. Default: ``True``
     divisor_override: if specified, it will be used as divisor, otherwise
          size of the pooling region will be used. Default: None
+
+Keyword args:
+    out: the output tensor. If :attr:`out` is used, this operation won't
```

Comment:
@albanD do you think we should we take this opportunity to explicitly mention in the docs that we expect the out= param to already be the right shape, or to be a zero-sized tensor? (idk if/when we are going to remove that deprecation warning)

Reply:
Ho yeah we could have either a generic message here or a full notes about "Semantic of the out= variants" that we can link to from all the docstrings. WDYT?